src/BizHawk.Emulation.Common/Interfaces/Services/IStatable.cs

```diff
@@ -18,11 +18,8 @@ public interface IBinaryStateable
 	/// If unavailable these options will not be exposed
 	/// Additionally many tools depend on savestates such as TAStudio, these will only be available if this service is implemented
 	/// </summary>
-	public interface IStatable : IEmulatorService
+	public interface IStatable : IBinaryStateable, IEmulatorService

```

Comment:
@adelikat `IStatable` used to inherit `IBinaryStateable` but then you changed it not to and instead copy+paste the method signatures.  I changed it back here without thinking too much; any idea what was up with that?  The two interfaces do have the exact same semantic meaning.

Reply:
It was my attempt to detangle them (eventually).  IStatable should be a BinaryStatable.  But I can detangle later.  If this makes for a cleaner waterbox class, go for it

Reply:
I don't see how anything is "entangled"?  We have one interface which does one universal thing throughout Bizhawk.